lib/services/registered_addons.pm

```diff
@@ -0,0 +1,83 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check registered system and addons status
+#
+# Maintainer: Yutao Wang <yuwang@suse.com>
+
+package services::registered_addons;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+use version_utils 'is_sle';
+use registration 'get_addon_fullname';
+
+sub zypper_lr {
```

Comment:
Could you support the output log for later debug? This can be done by upload a file or just print all info to the autoinst-log.txt